src/components/ContextBar.js

```diff
@@ -101,6 +113,17 @@ export const ContextBar = ({
     ])
   }
 
+  //This excludes cromwellapp in the calculation.
```

Comment:
Are you excluding Cromwell because cost calculations are not available for it? 

Ideally it would be nice to just iterate though each of the tools and ask them for their cost (and make it up to the tools to figure out their cost… Cromwell could just return 0 for now).